### PR TITLE
Oleksandr task#5 improve class car

### PR DESCRIPTION
**Pull request by Sasha1152**

### CODE REVIEW

**Comment by Sasha1152** [comment]
Done

File: Oleksandr_Shmigelskiy/task05_class_auto_improvement.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        return self.model
+
+    def get_year(self):
+        return self.year
+
+    def get_max_speed(self):
+        return self.max_speed
+
+    def set_car_type(self, car_type_needed):
+        for subclass in Car.__subclasses__():
```
Comment (pivancy) [inline Oleksandr_Shmigelskiy/task05_class_auto_improvement.py]: Please don't use magic methods directly.

File: Oleksandr_Shmigelskiy/task05_class_auto_improvement.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            temp_object = subclass(self.model, self.year, self.max_speed)
+            if car_type_needed.title() == temp_object.__dict__['car_type']:
+                self = temp_object
```
Comment (pivancy) [inline Oleksandr_Shmigelskiy/task05_class_auto_improvement.py]: It's a super bad idea to override `self`.

**Comment by pivancy** [comment]
Left group 2 days ago!

**Pull request closed without merging**